### PR TITLE
[Merged by Bors] - chore(Data/Matrix): some trivial lemmas about `row` and `col`

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Matrix/RowCol.lean [hunk 1]
```diff
@@ -136,6 +136,16 @@ theorem row_mulVec [Fintype n] [NonUnitalNonAssocSemiring α] (M : Matrix m n α
   ext
   rfl
 
+theorem row_mulVec_eq_const [Fintype m] [NonUnitalNonAssocSemiring α]  (v w : m → α) :
+    Matrix.row ι v *ᵥ w = Function.const _ (v ⬝ᵥ w) := rfl
+
+theorem mulVec_col_eq_const [Fintype m] [NonUnitalNonAssocSemiring α] (v w : m → α) :
+    v ᵥ* Matrix.col ι w = Function.const _ (v ⬝ᵥ w) := rfl
+
+theorem row_mul_col_eq [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```
Comment (YaelDillies) [inline Mathlib/Data/Matrix/RowCol.lean:145]:
```suggestion
theorem row_mul_col_eq_const [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```
no?

**Review by YaelDillies — COMMENTED** [review]
Should these lemmas be simp?

File: Mathlib/Data/Matrix/RowCol.lean (hunk 1 above)
Comment (eric-wieser) [reply Mathlib/Data/Matrix/RowCol.lean:145]: The RHS doesn't mention `Function.const`

**Comment by eric-wieser** [comment]
> Should these lemmas be simp?

I'd argue no, because there isn't really any matrix api around `Function.const`, so that's more likely to take you into a dead end.

File: Mathlib/Data/Matrix/RowCol.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Data/Matrix/RowCol.lean:145]: Then I'd suggest removing the `_eq`

Comment (eric-wieser) [reply Mathlib/Data/Matrix/RowCol.lean:145]:
```suggestion
theorem row_mul_col [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```

**Comment by YaelDillies** [comment]
maintainer merge

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**